README.md

```diff
@@ -113,9 +113,9 @@ Using the following command only the Identity Service and the [nginx-ingress](ht
 helm repo add alfresco-stable https://kubernetes-charts.alfresco.com/stable
 
 helm install alfresco-stable/alfresco-infrastructure \
-  --set alfresco-infrastructure.activemq.enabled=false \
-  --set alfresco-infrastructure.nginx-ingress.enabled=true \
-  --set alfresco-infrastructure.alfresco-identity-service.enabled=true \
+  --set activemq.enabled=false \
+  --set nginx-ingress.enabled=true \
+  --set alfresco-identity-service.enabled=true \
```

Comment:
This should be in accordance with the conditions set on the infrastructure chart. https://github.com/Alfresco/alfresco-infrastructure-deployment/blob/master/helm/alfresco-infrastructure/requirements.yaml

Reply:
Fixed all values to follow the `requirements.yaml` conditions.